packages/worker/src/transcribe.ts

```diff
@@ -361,7 +361,7 @@ export const runWhisper = async (
 		const result = await runSpawnCommand('transcribe', 'docker', [
 			'exec',
 			containerId,
-			'whisper.cpp/main',
+			'whisper.cpp/build/bin/whisper-cli',
```

Comment:
this is updated following the docs here https://github.com/ggerganov/whisper.cpp